docs/_plugins/layer-switching-controls/l-switchasemap.md

```diff
@@ -0,0 +1,12 @@
+---
+name: L.switchBasemap
+category: layer-switching-controls
+repo: https://github.com/clavijojuan/L.switchBasemap
+author: clavijojuan
+author-url: https://github.com/clavijojuan
+demo: https://elegant-meninsky-515912.netlify.app/
```

Comment:
I'm against merging this PR until attribution is displayed in the demo. Tile hosters want to have a copyright notice while usage: https://www.openstreetmap.org/copyright

The attribution should also changed while switching the base layer.

![grafik](https://user-images.githubusercontent.com/19800037/151411105-7e914137-d154-477a-836e-6268b4119357.png)

Reply:
Hi, fixed. Sorry for that.